### ISSUE TITLE
Perspective controls are messed

### ISSUE THREAD

**Issue by krakozavr** [rsjaffe/MIDI2LR#107]
Horizontal - Verticacal perspective controls are messed.
I have both assigned to knobs. 
Now I found that the Vertical control slider moving the Horisontal Perspective control in LR, and the Horizontal control slider moving the Vertical control. But only for PORTRAIT photos; for LANDSCAPE photos everything works as expected.

MIDI2LR 8.0.1, LRCC, WIN10; it's for SONY A7RII RAWs (ARWs) only (for JPEGs everythings works as expected). Probably it's some bug with ARW but I'm not sure :)

**Comment by rsjaffe** [comment]
Sounds like a LR bug. If JPGs work fine in both orientations but ARW not, it's like LR doesn't know it has rotated the image. Have you converted a couple of ARWs to DNG in portrait and landscape mode to check? You might want to notify Adobe of the issue.

**Comment by krakozavr** [comment]
I just converted one ARW into DNG and this bug still persist with it. 
Also, I checked my CRW from Canon 5D-II and I found the same issue for portrait pictures. 

I'm not sue how it was worked with the preview versions of MIDI2LR; but I checked some portrait RAWs which was edited with 0.7.0 and I found there perspective corrections. I don't remember if I have had this issue there. Probably somebody who still using 0.7.0 can check it?
